source/index.html

```diff
@@ -1,12 +1,182 @@
+    <p class="page__body-intro">Занялся собой? Займись котом!</p>
+    <a class="button button__for-program page__body-button" href="program.html">Подобрать программу</a>
+    <!-- логика: общий класс-блок для кнопки, класс-элемент для видоизменения кнопки, класс-элемент для расположения кнопки. правильно ли так рассуждать? -->
+    <section class="packs">
+      <h2 class="visually-hidden">Варианты программ</h2>
+      <ul class="packs__list">
+        <li class="packs__item packs__item--slim pack">
+          <h3 class="pack__title">Похудение</h3>
+          <p class="pack__description">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора
+            на диету! Cat Energy Slim поможет
+            вашему питомцу сбросить лишний вес.
+          </p>
+          <a class="pack__button" href="#">
+            <!-- вот тут я так понимаю нужен еще один класс, т.к. действие по клику будет разное, но может ли быть feachure__button-slim ? -->
+            Каталог slim
+            <svg class="pack__button-icon"></svg>
+          </a>
+        </li>
+        <li class="packs__item packs__item--pro pack">
+          <h3 class="pack__title">Набор массы</h3>
+          <p class="pack__description">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro
+            поможет вашему коту нарастить необходимые
+            мышцы!
+          </p>
+          <a class="pack__button" href="#">
+            <!-- аналогично. вот тут я так понимаю нужен еще один класс, т.к. действие по клику будет разное, но может ли быть feachure__button-pro ? -->
+            Каталог pro
+            <svg class="pack__button-icon"></svg>
+          </a>
+        </li>
+      </ul>
+    </section>
+    <section class="feachures">
+      <h2 class="feachures__title">Как это работает</h2>
+      <ol class="feachures__list">
+        <li class="feachures__item">
+          <span class="feachures__icon feachures__icon--leaf"><span class="visually-hidden">Иконка листок</span></span>
+          <p> Функциональное питание
+            содержит только полезные
+            питательные вещества.
+          </p>
+        </li>
+        <li class="feachures__item">
+          <span class="feachures__icon feachures__icon--water"><span class="visually-hidden">Иконка стакан
+              воды</span></span>
+          <p> Выпускается в виде порошка,
+            который нужно лишь залить
+            кипятком и готово.
+          </p>
+        </li>
+        <li class="feachures__item">
+          <span class="feachures__icon feachures__icon--food"><span class="visually-hidden">Иконка столовые
+              приборы</span></span>
+          <p> Замените один-два приема
+            обычной еды на наше
+            функциональное питание.
+          </p>
+        </li>
+        <li class="feachures__item">
+          <span class="feachures__icon feachures__icon--time"><span class="visually-hidden">Иконка часы</span></span>
```

Comment:
Кстати, всегда рекомендую при наличии вложенного тега давать новую строку.
Не то чтобы это важно, но считывается легче =)
```suggestion
          <span class="feachures__icon feachures__icon--time">
            <span class="visually-hidden">Иконка часы</span>
          </span>
```